README.md

```diff
@@ -40,19 +56,19 @@ If you want to submit a new feature or a bugfix, the best way is to create the c
   * The top of each Markdown file is a block of YAML for page specific localization information that is passed to various templates.
   * The bulk of the Markdown content for each page is referenced as `{{{content}}}` in the corresponding template.
 
-### Serve/Build Options
+## Serve/Build Options
```

Comment:
I think these were all `###` before because there are intended to be nested under `## Contributing`